test/vol.cpp

```diff
@@ -399,6 +415,24 @@ int main(const int argc, const char** argv)
                  //<<usage.vsize
                  <<std::endl;
 	}
+	if(exper){
```

Comment:
If a test fails then the program should stop or return some error status, right? Otherwise, how do you know if some tests are failing or not?